__tests__/app.test.js

```diff
@@ -663,3 +663,43 @@ describe("12. DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("13. GET /api", () => {
+  it("status:200, should responds with an array of topics", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(
+          expect.objectContaining({
```

Comment:
I would probably just check for an object, or if you really want, the keys - otherwise you're going to have to keep adding to this when you update the file - but its up to you